person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, rentals, name = 'Unknown', _parent_permission: true)
```

Comment:
- [ OPTIONAL ] rentals does not need to be included here, it just needs to be initialized down there as you already did